docs/localizability.md

```diff
@@ -24,9 +24,11 @@ Unfortunately, NuGet.Client does not use OneLocBuild, and any resource comments
 Although the steps below are for the dev branch, the process is similar for release branches.
 
 1. A PR merges into the dev branch.
-1. A NuGet.Client build generates a `localizationArtifacts` artifact.  This artifact contains newly built (but unsigned) assemblies and .lcg files.  The .lcg files contain information extracted from .resx files and are the primary input for later localization.
+1. A NuGet.Client build generates a `localizationArtifacts` artifact.  This artifact contains newly built Authenticode-signed assemblies [(see PR#6170)][12]) containing English resources and .lcg files.
+The .lcg files contain information extracted from .resx files and are the primary input for later localization.
 1. The localization team grabs the `localizationArtifacts` artifact from a NuGet.Client build.
-1. The localization team localizes strings and merges a ["LEGO" PR][5] with localized strings (in .lcl files).
+1. The localization team localizes strings.
+The signed assemblies are used to validate the exported .lcl won't break the build before the team merges a ["LEGO" PR][5] with localized strings (in .lcl files).
```

Comment:
```suggestion
The assemblies are used to validate the exported .lcl won't break the build before the team merges a ["LEGO" PR][5] with localized strings (in .lcl files).
```

Reply:
Consolidating to thread: https://github.com/NuGet/NuGet.Client/pull/6172#discussion_r1866718472